bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -399,4 +399,18 @@ export class SequencesPage {
     cy.byTestId('keptn-blocking-sequence-navigation').click();
     return this;
   }
+
+  public hoverOverStageStartTime(stage: string): this {
+    cy.byTestId(`${stage}StartTime`).trigger('mouseenter');
+    return this;
+  }
+
+  public assertTimeOverlay(status = true): this {
+    const overlay = cy.get('.cdk-overlay-container');
+    overlay.should(status ? 'exist' : 'not.exist');
+    if (status) {
+      overlay.children().should('have.length.greaterThan', 0);
```

Comment:
this check only looks for a string with lenght > 0, right? 
I would suggest, if possible, to verify for the regex `[0-9]{4}-[0-9]{2}-[0-9]{2}\s[0-9]{2}:[0-9]{2}:[0-9]{2}`